allennlp/data/fields/metadata_field.py

```diff
@@ -61,7 +62,7 @@ def empty_field(self) -> 'MetadataField':
     @classmethod
     @overrides
     def batch_tensors(cls, tensor_list: List[DataArray]) -> DataArray:  # type: ignore
-        return tensor_list  # type: ignore
+        return ScatterableList(tensor_list)  # type: ignore
```

Comment:
The `ProductionRuleField` needs a similar modification.

Reply:
Good catch, thanks! This does make me wonder if we should modify the type that `batch_tensors` is supposed to return to include the possibility of using `ScatterableList`. But historically we've just been ignoring types here. My intuition for this in Python is not particularly developed.